src/main/java/org/sonarlint/intellij/trigger/SonarLintSubmitter.java

```diff
@@ -73,45 +73,42 @@ public void submitFilesModal(Collection<VirtualFile> files, TriggerType trigger)
   }
 
   public void submitFilesModal(Collection<VirtualFile> files, TriggerType trigger, @Nullable AnalysisCallback callback) {
-    Multimap<Module, VirtualFile> filesByModule = filterAndgetByModule(files, false);
+    Map<Module, Collection<VirtualFile>> filesByModule = filterAndgetByModule(files, false);
 
     if (!filesByModule.isEmpty()) {
       console.debug("Trigger: " + trigger);
 
       for (Module m : filesByModule.keySet()) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused "m" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1481)